CONTRIBUTING.md

```diff
@@ -82,17 +89,24 @@ Your pull request should be rebased against the current master branch. Please do
 the current master branch in with your topic branch, nor use the Update Branch button provided
 by GitHub on the pull request page.
 
+## Submitting a Plugin
+
+Sharing a new plugin is one of the most impressive ways to jump into the Snap community as a contributor. This process is as simple as adding your plugin information to the [list in plugis.yml](docs/plugin.yml). Read more about what is required of plugin authors in [Plugin Authoring](docs/PLUGIN_AUTHORING.md).
```

Comment:
I mean "most impressive" as a way to prompt people to recognize our appreciation for plugins. Plugin design is pretty straightforward, but it's by far not the easiest way to get involved. That would be through opening issues, tweeting about using it, blogging, etc.